FishjamClient/src/test/java/fishjam/client/WebsocketMock.kt

```diff
@@ -1,7 +1,7 @@
-package jellyfishdev.jellyfishclient
+package fishjamdev.fishjamclient
```

Comment:
nit:
```suggestion
package fishjamdev.client
```
(to partially match code package name)

Reply:
👍